.travis.yml

```diff
@@ -6,9 +6,12 @@ dist: trusty
 
 matrix:
   include:
-  - name: "PHP 7.2 unit tests, PHP Coding standards check and JS tests"
-    php: 7.2
+  - name: "PHP 7.3 unit tests, PHP Coding standards check and JS tests"
+    php: 7.3
     env: WP_VERSION=latest WP_MULTISITE=0 WP_CORE_DIR=/tmp/wordpress RUN_PHPCS=1 RUN_JS=1
+  - name: "PHP 7.2 unit tests"
+    php: 7.2
+    env: WP_VERSION=latest WP_MULTISITE=0 WP_CORE_DIR=/tmp/wordpress
```

Comment:
Would passing `RUN_PHPCS=1` as an argument here work or do we need the `$TRAVIS_PHP_VERSION == "7.2"` below?

Reply:
> Would passing RUN_PHPCS=1 as an argument here work

Yes, that would work but run the code sniffer twice. The WC sniffs are a couple versions behind core so I'm going to look at a separate PR to see if we can get our config more in line with WC core.

Reply:
@joshuatf Thanks for the feedback. I'm putting this on hold pending #3571 being merged.

Reply:
✅ on https://github.com/woocommerce/woocommerce-admin/pull/3571 - if you want to rebase this one and ping me, I'll take another look.

Reply:
@joshuatf Thanks for the suggestion. I added a `COMPOSE_DEV` variable to signal using package PHPUnit.